website/src/main/java/com/google/step/data/CloudSQLManager.java

```diff
@@ -109,4 +109,31 @@ public void createTable(String tableName, List<String> columns) throws SQLExcept
     }
   }
 
+  public ResultSet getOrgsWithNeighbors(String keyword, int offset) throws SQLException{ 
+    String similarTo = (!keyword.isEmpty()) ? 
+        "WHERE (name LIKE '%" + keyword + "%' OR about LIKE '% " + keyword + "%' OR class ='" + keyword + "')" 
```

Comment:
Should `class = keyword` or should `class LIKE '%keyword%'`?

Reply:
So I made a mistake when writing the classification code where it includes classifications with less accuracy. 
So something could be Health/Diseases/Cancer and People/Livilhood or something like that. I fixed it but until we redo the database using class = keyword won't yield anything